corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/views.js

```diff
@@ -121,6 +121,7 @@ hqDefine("cloudcare/js/formplayer/apps/views", [
 
         initialize: function (options) {
             this.shouldShowIncompleteForms = options.shouldShowIncompleteForms;
+            sessionStorage.removeItem('handledDefaultClosed');
```

Comment:
Does this mean if you reload the page the setting will be lost?

Reply:
If you lock the nav menu open, you can reload the page and the menu will stay open. The nav menu will stay expanded until you unlock it or navigate out of the app to the home menu.